mu-plugins/blocks/global-header-footer/blocks.php

```diff
@@ -338,7 +338,9 @@ function render_global_header( $attributes = array() ) {
 
 	// Preload the menu font.
 	if ( is_callable( 'global_fonts_preload' ) ) {
-		global_fonts_preload( 'Inter latin' );
+		if ( ! is_rosetta_site() ) {
```

Comment:
Currently, the preloaded font is specified as `Inter Latin`, this would make the rosetta sites preload unnecessary font files. For example, https://ru.wordpress.org/ should only load the Cyrillic subsets, but without `if ( ! is_rosetta_site() ) {`, it would preload `Latin` as well. 

The issue was opened to fix it.